packages/theme/tokens/dimension.json

```diff
@@ -93,6 +93,50 @@
 						}
 					}
 				}
+			},
+			"gap": {
+				"flow": {
```

Comment:
What's the purpose of `flow`?

Reply:
I went with "flow" as a more generic, systems-level alternative to something like "stack", which would be very component-oriented, with the idea being that this relates to the _flow of content_. Although as I'm trying to explain it, I realize it may have went too far in the opposite direction of over-abstractedness 😅 I'm open to alternative, even open to "stack" for what it's worth.